### ISSUE TITLE
Add UI for observing outcome when deploying policies in "dry run" mode

### ISSUE THREAD

**Issue by nscuro** [DependencyTrack/hyades#959]
As part of https://github.com/DependencyTrack/hyades/issues/957, we identified that it would be good to have a "dry run" mode for policies. This mode would evaluate policies, but not apply their associated action.

Initially, this functionality is implemented using application logs. However, long-term, we can't expect users to observe logs. We should have a UI integration that provides this information instead.

The granularity of the information being displayed is still subject for discussion. But generally, the following would be nice to have:

* Number of affected projects
* Number of affected components
* Number of affected findings
  * Potentially with additional aggregates, as in "avg. findings affected per component / project"

**Comment by skhokhlov** [comment]
I'm thinking maybe DT shouldn't apply new bundle of policies automatically after pulling, but wait for review?

**Comment by VinodAnandan** [comment]
It's a good idea, but keeping it as the default behaviour while still allowing overrides makes it useful for multiple scenarios. Some may prefer to maintain a strict review and approval process solely within Git, without using multiple systems (Git and DT) for policy approval.

**Comment by skhokhlov** [comment]
Right, this may be configurable Sergey KhokhlovOn 15 Dec 2023, at 7:36 pm, Vinod Anandan ***@***.***> wrote:﻿
It's a good idea, but keeping it as the default behaviour while still allowing overrides makes it useful for multiple scenarios. Some may prefer to maintain a strict review and approval process solely within Git, without using multiple systems (Git and DT) for policy approval.
